### ISSUE TITLE
Mac Menu Bar

### ISSUE THREAD

**Issue by donatj** [cratonica/trayhost#9]
I don't know if this is possible, certainly not with the current code. Nor do I know if this is within the intended scope of this library but I would love to have my app in the universal menu bar on the top right of my mac rather than the Dock.

**Comment by dmitshur** [comment]
I think this library already does _exactly that_.

See this screenshot of an app that uses [my fork of this library](https://github.com/shurcooL/trayhost):

![](https://cloud.githubusercontent.com/assets/1924134/8891878/8dd7a024-32ee-11e5-8eca-1994f8503094.png)

Is that what you're looking for?

**Comment by donatj** [comment]
It is exactly what I'm looking for… But all I am getting is as follows: 
(excuse my really bad icon, its just a PNG I had that happened to already be 64x64)

![screen shot 2017-03-10 at 5 59 05 pm](https://cloud.githubusercontent.com/assets/133747/23817771/55e58f22-05bb-11e7-9b7e-11bcc7618b07.png) 

![image](https://cloud.githubusercontent.com/assets/133747/23817767/4e90da56-05bb-11e7-95e8-60ad56bc1be5.png)


with nothing in my status bar…


![screen shot 2017-03-10 at 5 59 00 pm](https://cloud.githubusercontent.com/assets/133747/23817781/65d71b12-05bb-11e7-97df-e9d4539a74cd.png)

**Comment by dmitshur** [comment]
I see. I guess this package has changed since I last looked at it (or maybe you're not using it in a way that gives you a tray icon, or using a very outdated version).

Try [my fork](https://github.com/shurcooL/trayhost), it should do what you want.

**Comment by cratonica** [comment]
Indeed, I think many of the forks are more feature-complete than this is now.
